crypto/edhoc-crypto-psa/src/lib.rs

```diff
@@ -388,13 +385,13 @@ mod rust {
         let my_key = key_management::import(attributes, None, &key[..]).unwrap();
         let mut output_buffer: [u8; PLAINTEXT_3_LEN] = [0; PLAINTEXT_3_LEN];
 
-        let (plaintext, err) =
-            match aead::decrypt(my_key, alg, iv, ad, ciphertext, &mut output_buffer) {
-                Result::Ok(_) => (output_buffer, EDHOCError::Success),
-                Result::Err(_) => ([0x00u8; PLAINTEXT_3_LEN], EDHOCError::MacVerificationFailed),
-            };
+        let ret = aead::decrypt(my_key, alg, iv, ad, ciphertext, &mut output_buffer);
```

Comment:
@malishav maybe `match` could be used here? e.g. as follows:

```
        match aead::decrypt(
            my_key,
            alg,
            &iv.to_public_array(),
            &ad.to_public_array(),
            &ciphertext.to_public_array(),
            &mut output_buffer,
        ) {
            Ok(_) => Ok(BytesPlaintext3::from_public_slice(&output_buffer[..])),
            Err(_) => Err(EDHOCError::MacVerificationFailed),
        }
```

Reply:
good point, fixed this occurence

Reply:
Great, maybe could also be changed in `decrypt_ccm`.